### DISCUSSION TITLE
2022 Wish list

### DISCUSSION THREAD

**Discussion by maxperkin1989** [simple-login/app#731] [Feature Requests]
Hello there.

I am loving your service but for the money that I pay, I would like to see more domains available that don't look so suspicious. 

I understand we can add our own but that is an additional cost to buy/maintain that domain. 

Please in 2022 can you add/update your premium domains that just look like regular email domains something like @xyztech.com @x2-mail.com etc

Can we please have an easier way to reply to our alias emails?

also is it possible to have haveibeenpwned support. that would be so convenient be able to see it from your portal 👍 

thanks for everything!

**Comment by nguyenkims** [comment]
Hey haveibeenpwned is already integrated, you only see it if your alias appears in a data leak though. 

> Can we please have an easier way to reply to our alias emails?

Currently it's quite easy to reply to a forwarded email, maybe you mean sending emails from an alias?

> Please in 2022 can you add/update your premium domains that just look like regular email domains something like @xyztech.com @x2-mail.com etc

Sometimes we buy new domains when a short, easy to remember one become available. If you notice a nice domain available, please send it to us so we can buy it :).

**Reply by Vnzq4Qje3zzU** [reply]
> If you notice a nice domain available, please send it to us so we can buy it :).

Please buy sl.me !!! It would be like pm.me from ProtonMail and it would be very convenient for reverse-alias addresses because of the readability and 64 characters limit.

**Reply by nguyenkims** [reply]
sl.me costs $25000 which is a bit too expensive for us :)

**Reply by c0nfigurati0n** [reply]
Lmao, yeah that is indeed very expansive. I wish it wasn't tho. I guess you have to do your research before you suggest a domain.

**Reply by ghost** [reply]
> sl.me costs $25000 which is a bit too expensive for us :)

Hey @nguyenkims , sent you an email with some cool short and cheap domains

**Reply by nguyenkims** [reply]
@diogobs yeah I've received it, thanks!

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
> sl.me costs $25000 which is a bit too expensive for us :)

Just an idea if u really want the domain. U could try to crowdfund the domain. Now that ur teaming up with Proton ur repo/service will get more traction/visitors/views.
